Document/0x06a-Platform-Overview.md

```diff
@@ -167,3 +167,17 @@ The following APIs [require user permission](https://www.apple.com/business/docs
 - Bluetooth sharing
 - Media Library
 - Social media accounts
+
+### DeviceCheck
+
+The DeviceCheck framework, including its components DeviceCheck and App Attest, helps you prevent fraud in promotional offers. It allows you to persistently store information on the device and on Apple servers to limit promotions to once per device. The stored information remains intact across app reinstallation, device transfers, and even resets. You can reset this data periodically.

```

Comment:
I've enhanced it a little bit to put more emphasis on the real-world use cases, what do you think?

```suggestion
The DeviceCheck framework, including its components DeviceCheck and App Attest, helps you prevent fraudulent use of your services. It consists of a framework that you use from your app and an Apple server which is accessible only to your own server. DeviceCheck allows you to persistently store information on the device and on Apple servers. The stored information remains intact across app reinstallation, device transfers, or resets, with the option to reset this data periodically.

DeviceCheck is typically used to mitigate fraud by restricting access to sensitive resources. For example, limiting promotions to once per device, identify and flag fraudulent devices, etc. However, it definitely cannot prevent all fraud. For example, is not meant to detect compromised operating systems (aka. jailbreak detection). 
```